spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkFilesScan.java

```diff
@@ -32,6 +32,8 @@
 import org.apache.iceberg.util.TableScanUtil;
 import org.apache.spark.sql.SparkSession;
 
+/** @deprecated use {@link SparkStagedScan} instead */
```

Comment:
Such annotations should also include the version in which we plan to remove it.

```
/** @deprecated will be removed in 1.3.0, use {@link SparkStagedScan} instead */
```

Reply:
Done

Reply:
Looks like this was missed.

Reply:
Whoops, fixed.